ui/admin/app/routes/utils/setupFilters.js

```diff
@@ -0,0 +1,4 @@
+export function setupFilters(controller, route, queryParam) {
+  const item = controller.controllerFor(route)[`filter-${queryParam}`];
+  return item ? JSON.parse(item) : null;
```

Comment:
Is this return working?

Assuming `item` is a `string` or `object`. I recreate this return with an `if` statement and did not work for me, maybe I am missing something.